meteor-app/imports/ui/Report_View/Report_View.tsx

```diff
@@ -9,7 +9,7 @@ export const Report_View = () => {
 	const [report, setReport] = useState(null)
 
 	const makeReport = () => {	
-		Meteor.call('Compose_Report', id, (error, result) => {
+		Meteor.call('Compose_Report', id, (error : any, result : any) => {
```

Comment:
the proper way to type the `error` object is to use `error: Error`

for the result, you would use the interface `result: ReportStructure`